test/strategy.coffee

```diff
@@ -129,7 +134,7 @@ describe 'EVE Online OAuth Strategy', ->
         @oAuth2VerifyDone)
 
     it 'translates passport-oauth2 verifications', ->
-      @verify.should.be.calledWith(@profile, @oAuth2VerifyDone)
+      @verify.should.be.calledWith('access token', 'refresh token', @profile, @oAuth2VerifyDone)

```

Comment:
It's passed passport-oauth2 `super(options, @_verifyOAuth2)` then invoked here:
https://github.com/jaredhanson/passport-oauth2/blob/master/lib/strategy.js#L191-L193